linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages/ansible/shared.yml

```diff
@@ -1,16 +1,12 @@
-# platform = Red Hat Enterprise Linux 8,multi_platform_ol,multi_platform_almalinux
+# platform = Red Hat Enterprise Linux 8,multi_platform_almalinux,multi_platform_ol,multi_platform_sle
 # reboot = true
 # strategy = restrict
 # complexity = low
 # disruption = low
 
-- name: Ensure xorg packages are removed
+{{% for package in xwindows_packages %}}
+- name: Ensure {{{ package }}} is removed
```

Comment:
You need to add `{{{ rule_title }}} - ` to the Ansible Task's name.

Reply:
Done in [6878d53](https://github.com/ComplianceAsCode/content/pull/12853/commits/6878d5395e2a3e4ceba9969c4fd06dc739db32fa) :bow: